### PR TITLE
Add monitor in ModelCheckpoint 

### PR DESCRIPTION
**Pull request by Jerry-Jzy**
## what is the purpose of this pull request
Add monitor in ModelCheckpoint

### CODE REVIEW

File: deepxde/nn/pytorch/mionet.py [hunk 1]
```diff
@@ -186,4 +186,4 @@ def forward(self, inputs):
             y += self.b
         if self._output_transform is not None:
             y = self._output_transform(inputs, y)
-        return y
+        return y
```
Comment (lululxvi) [inline deepxde/nn/pytorch/mionet.py:189]: Do not change this line.

File: deepxde/callbacks.py [hunk 2]
```diff
@@ -117,16 +117,24 @@ class ModelCheckpoint(Callback):
             monitored. Model is only checked at validation step according to
             ``display_every`` in ``Model.train``.
         period: Interval (number of epochs) between checkpoints.
+        monitor: The loss function that is monitored. Either 'train loss' or 'test loss'
```
Comment (lululxvi) [inline deepxde/callbacks.py:120]: Add "." in the end.

File: deepxde/nn/pytorch/mionet.py (hunk 1 above)
Comment (Jerry-Jzy) [reply deepxde/nn/pytorch/mionet.py]: fixed

File: deepxde/callbacks.py (hunk 2 above)
Comment (Jerry-Jzy) [reply deepxde/callbacks.py]: Fixed

Comment (lululxvi) [reply deepxde/callbacks.py]: 'test loss'.

Comment (Jerry-Jzy) [reply deepxde/callbacks.py]: sorry about the careless

**Pull request merged**